manual_flagging/manual_update.py

```diff
@@ -186,23 +186,33 @@
     condos=True,
 )
 
-df_flagged_merged = pd.concat(
-    [df_res_flagged_updated, df_condo_flagged_updated]
-).reset_index(drop=True)
+# Update the PTAX flag column with an additional std dev conditional w/ res groups
+df_res_flagged_updated_ptax = flg.ptax_adjustment(
+    df=df_res_flagged_updated,
+    groups=inputs["stat_groups"],
+    ptax_sd=inputs["ptax_sd"],
+    condos=False,
+)
```

Comment:
**[Thought, non-blocking]** It's a bummer that to change any of the flagging logic, you have to make nearly identical changes in three different scripts; not a great practice for long-term maintainability. Once this pipeline gets to a stable state, we should brainstorm ways that we can centralize the flagging logic.